auth0/resource_auth0_action_test.go

```diff
@@ -179,3 +198,74 @@ resource auth0_action my_action {
 	}
 }
 `
+
+func TestCheckForUntrackedActionSecrets(t *testing.T) {
```

Comment:
These test cases are sensible IMO. However, I'm noticing that there are no cases where `givenOldSecretsConfig` is not empty, which solidifies my point above that the separation of old and new isn't needed.